code/mathematical_algorithms/src/newman_conway/README.md

```diff
@@ -0,0 +1,20 @@
+# Newman-Conway Number  
```

Comment:
Make changes to follow [this guide](https://github.com/OpenGenus/cosmos/blob/master/guides/documentation_guide.md). Otheriwse, it's good!

Reply:
Can you suggest me the changes?

Reply:
It's quite simple, just take the template, copy it into your Markdown, and fill in all required information.